yelp_package/dockerfiles/itest/chronos/Dockerfile

```diff
@@ -14,20 +14,20 @@
 
 FROM ubuntu:trusty
 
-RUN apt-get update && apt-get -y install apt-transport-https
+RUN DEBIAN_FRONTEND=noninteractive apt-get update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -y install apt-transport-https > /dev/null
```

Comment:
I think they are silly to actively discourage ENV variables like that. They are technically correct, but these are not docker images that people are pulling from around the world or anything, they are our images.

Please just do it to make it cleaner. I accept the caveat that anyone using these docker images in an interactive way will not get apt-get output. Fine :)